example/spec/parallel/ingestGranule/IngestGranuleFailureSpec.js

```diff
@@ -168,6 +163,17 @@ describe('The Ingest Granule failure workflow', () => {
     });
 
     it('logs the execution with the error message', async () => {
+      await waitForModelStatus(
```

Comment:
We might want to consider moving this to beforeAll for explicitness/in case of future us needing it for another test. 

That said, this looks great!